recipes/arraylake/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "arraylake" %}
+{% set version = "0.7.2a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/arraylake-{{ version }}.tar.gz
+  sha256: d5674aac64c3aac2c02b0ed38d250c0c9d30c37c8428f359d39cb2e9269370a5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - arraylake = arraylake.cli.main:app
+
+requirements:
+  host:
+    - python >=3.8,<3.12
+    - poetry-core >=1.0.0
+    - poetry-dynamic-versioning
+    - pip
+  run:
+    - python >=3.8,<3.12
+    - aiobotocore >=2.4,<3
+    - zarr >=2.14,<=2.16
+    - numpy >=1.20,<2
+    - donfig >=0.7,<1.0
+    - typing-extensions >=4.7.0,<5.0.0
+    - pydantic >=1.10,<2
+    - httpx >=0.23,<1.0
+    - cachetools >=5.3,<6
+    - structlog >=23.1,<24
+    - email_validator >=2,<3
+    - click >=8.1,<9
+    - rich >=12.6,<14.0
+    - typer >=0.6,<1.0
+    - pyyaml >=6,<7.0
+    - ruamel.yaml >=0.17,<1.0
+    - python-dateutil >=2,<3
+    - uvloop >=0.17,<1  # [unix]
+    - aioitertools >=0.11,<1
+  run_constrained:
+    - uvloop >=0.17,<1.0
+    - ruamel-yaml >=0.17,<1.0
+    - typer >=0.6.1,<1.0
+    - rich >=12.6,<14.0
+    - fsspec >=2022.11.0
+    - kerchunk >=0.1,<1.0
+    - h5py >=3.7.0,<4.0.0
+    - s3fs >=2022.11.0
+    - ipytree >=0.2.2,<1.0
+    - xarray >=v2022.12.0
+    - click >=8.1.0,<9.0.0
+
+test:
+  imports:
+    - arraylake
+  commands:
+    # - pip check
```

Comment:
Can you comment on what is failing?

Reply:
Ping

Reply:
We were having some trouble with `types-aiobotocore` earlier. We can try this again though as that is not a dependency anymore.

Reply:
Commenting it out is fine but we need to know so we can fix it in the dependencies.

Reply:
Looks like this was readded in recent changes and is in the feedstock